### ISSUE TITLE
Tab key support

### ISSUE THREAD

**Issue by enceeobee** [enceeobee/xwords#2]
We should support pressing the tab key, and it should behave like shift+arrow (i.e. 'skip'). This will improve navigation and accessibility.
